mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -1107,22 +1107,25 @@ bool CMomentumGameMovement::CheckJumpButton()
     if (g_pGameModeSystem->GameModeIs(GAMEMODE_RJ))
         mv->m_outStepHeight += 0.05f; // 0.15f total
 
-    // First do a trace all the way down to the ground
-    TracePlayerBBox(mv->GetAbsOrigin(),
-                    mv->GetAbsOrigin() + Vector(0.0f, 0.0f, -(sv_considered_on_ground.GetFloat() + 0.1f)),
-                    PlayerSolidMask(), COLLISION_GROUP_PLAYER_MOVEMENT, pm);
-
-    // Did we hit ground (ground is at max 2 units away so fraction cant be 1.0f)
-    if (pm.fraction != 1.0f && !pm.startsolid && !pm.allsolid)
+	if (!g_pGameModeSystem->GameModeIs(GAMEMODE_RJ))
```

Comment:
Convert tab to space here

Reply:
Resolved in commit d64284cae778dc917facf2b7426ca74761f2d0c6